app/locales/pt-PT.json

```diff
@@ -0,0 +1,188 @@
+{
+  "Postal code": "Código postal",
+  "Important": "Importante",
+  "Page not found": "Página não encontrada",
+  "We can't find the page or service you're looking for": "Não nos foi encontrar a página ou serviço que procura",
+  "The link you clicked may be broken, or the page no longer exists": "O endereço que abriu pode estar incompleto ou a página pode não existir",
+  "Something went wrong!": "Algo correu mal!",
+  "Something must have gone wrong": "Algo não está a funcionar corretamente",
+  "Send an e-mail to": "Envie um e-mail para",
+  "if you can't get it to work.": "se o problema persistir.",
+  "Before proceeding please note down the following link": "Antes de continuar por favor tome nota do seguinte endereço",
+  "Save the following link somewhere, or bookmark it in your browser.": "Anote o seguinte endereço ou guarde nos favoritos do seu browser.",
```

Comment:
This is just nitpicking, but I propose a few changes to make the translation more accurate with the original expression:

"Anote o seguinte endereço, ou guarde-o nos favoritos do seu browser."